server/paths.ts

```diff
@@ -106,3 +107,28 @@ export const getRedirects = () => {
 
   return result;
 };
+
```

Comment:
Nit: Let's add an example of generated config entry in comment here or a `LinkWithRedirect` type. Its hard to visualize content of the `map` right now.